scripts/_helpers.py

```diff
@@ -76,6 +77,14 @@ def handle_exception(exc_type, exc_value, exc_traceback):
         )
 
 
+def copy_default_files():
+    fn = Path("config.yaml")
+    if not fn.exists():
+        fn.write_text(
+            "# Write down config entries differing from config.default.yaml\n\nrun: {}"
+        )
```

Comment:
with an empty config.yaml the config.default.yaml will set everything (incremental config addition) so this will be the fallback case, if a user does not have any config.yaml file defined

Reply:
Right! yeah, you create as empty and just write that line.
Ok :) I should have seen it